### PR TITLE
Implement get nighlty on get build

### PR DESCRIPTION
**Pull request by ShakoHo**
@ypwalter , @askeing r?

### CODE REVIEW

**Comment by ShakoHo** [comment]
@MikeLien r?

File: tools/get_build.py [hunk 1]
```diff
@@ -1,35 +1,45 @@
 """
 
 Usage:
-  get_build.py <user_email> <platform> [--build-hash=<str>] [--no-status-check]
+  get_build.py <repo> <platform> [--user-email=<str>] [--build-hash=<str>] [--no-status-check] [--output-dp=<str>]
```
Comment (askeing) [inline tools/get_build.py]: is it possible to show `repo list` and `platform list` in usage content?

**Comment by askeing** [comment]
#319

**Comment by MikeLien** [comment]
lgtm r+, as askeing said, show repo list and platform list in usage content may be better.

**Pull request merged**